packages/search/search.test.ts

```diff
@@ -313,6 +313,7 @@ describe('search prompt', () => {
     await Promise.resolve();
     expect(getScreen()).toMatchInlineSnapshot(`
       "? Select a country France
+       ~ Americas ~
```

Comment:
Not sure why this didn't show before, but this outcome like a more consistent behaviour. Looks like a bug in previous implementation.